### PR TITLE
Add link to Tephra User group, and descriptions of the groups.

### PR DESCRIPTION
**Pull request by awholegunch**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -602,11 +602,21 @@ Bug Reports & Feature Requests
...
 Bugs and tasks are tracked in a public JIRA `issue tracker <https://issues.cask.co/browse/TEPHRA>`__.
 
-Mailing List
-............
+Tephra User Groups and Mailing Lists
+....................................
+
+- Tephra User Group: `tigon-user@googlegroups.com <https://groups.google.com/d/forum/tigon-user>`__
+
+  The *tigon-user* mailing list is primarily for users using the product to develop
```
Comment (poornachandra) [inline README.rst]: Tigon?

**Comment by poornachandra** [comment]
@awholegunch Looks like you made the update in the wrong repo. This change should probably go to tigon repo.

**Comment by awholegunch** [comment]
No, just a bad copy 'n paste...

**Comment by poornachandra** [comment]
LGTM

**Pull request merged**